src/resources/help/en_US/relnotes3.3.0.html

```diff
@@ -39,6 +39,7 @@
   <li>Default behavior of Python's <code>help(thing)</code> was adjusted slightly to first present what <code>help(thing)</code> would normally produce but then to also follow it up with output from <code>apropos(thing)</code> when that output is non-empty.<li>
   <li>The French translation was updated thanks to <a href="https://github.com/cessenat">@cessenat</a>. We are still looking for volunteers for translations in other languages. If interested, please <a href="https://visit-dav.github.io/visit-website/support/">contact us</a>.</li>
   <li>A new (beta) <a href="https://en.wikipedia.org/wiki/Names_given_to_the_Spanish_language">Castilian Spanish</a> translation was created thanks to <a href="https://github.com/cessenat">@cessenat</a> using auto-translation from English. It likely still requires review by someone who is fluent in Spanish and we would welcome any volunteers.</li>
+  <li>The Pick operator now has the option to include the Euclidean distance between the current and previous Pick in the GUI's output window.</li>
```

Comment:
Could add more sentences that describe how it behaves and what the controls are on the pick window?

Reply:
Will do.

Reply:
I've updated this to better describe the controls and output.